scripts/install-flux.sh

```diff
@@ -5,10 +5,11 @@ ENV=$1
 CLUSTER_NAME=$2
 VAULT_NAME=$3
 ENABLE_HELM_TLS=$4
-GIT_REPO=https://weaveworks.github.io/flux
-VALUES=deployments/weave-flux/values.yaml
+HELM_REPO=https://fluxcd.github.io/flux
+VALUES=deployments/fluxcd/values.yaml
+FLUX_HELM_CRD=deployments/fluxcd/flux-helm-release-crd.yaml
```

Comment:
Any reason for not pointing here : https://raw.githubusercontent.com/fluxcd/flux/master/deploy-helm/flux-helm-release-crd.yaml

Reply:
Figured it was safer to include it in our repo, and more reliable if github / w/e is glitching.
I can't see anywhere else where we're loading remote manifests

Reply:
I am only thinking that we need to be updating it whenever they change , where as helm chart is taken from their github. But its CRD, not hoping it to change often.